source/user-manual/agents/remote-upgrading/wpk-list.rst

```diff
@@ -31,3 +31,17 @@ macOS
++==============+=========+==============+=============================================================================================================================================================================================================================================================================================+
+|    Linux     |  |WAZUH_LATEST|  |     64bit    | `wazuh-agent-linux-x86_64-debug-info-v|WAZUH_LATEST|-|WAZUH_REVISION_WPK_AGENT_LINUX|.tar.gz <https://packages.wazuh.com/4.x/wpk/linux/x86_64/wazuh-agent-linux-x86_64-debug-info-v|WAZUH_LATEST|-|WAZUH_REVISION_WPK_AGENT_LINUX|.tar.gz>`_ (`sha512 <https://packages.wazuh.com/|CURRENT_MAJOR|/checksums/wazuh/|WAZUH_LATEST|/wazuh-agent-linux-x86_64-debug-info-v|WAZUH_LATEST|-|WAZUH_REVISION_WPK_AGENT_LINUX|.tar.gz.sha512>`__) |
++--------------+---------+--------------+---------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------+
+|    Windows   |  |WAZUH_LATEST|  |    32/64bit  | `wazuh-agent-windows-x86_64-debug-info-v|WAZUH_LATEST|-|WAZUH_REVISION_WPK_AGENT_WINDOWS|.zip <https://packages.wazuh.com/4.x/wpk/windows/wazuh-agent-windows-x86_64-debug-info-v|WAZUH_LATEST|-|WAZUH_REVISION_WPK_AGENT_WINDOWS|.zip>`_ (`sha512 <https://packages.wazuh.com/|CURRENT_MAJOR|/checksums/wazuh/|WAZUH_LATEST|/wazuh-agent-windows-x86_64-debug-info-v|WAZUH_LATEST|-|WAZUH_REVISION_WPK_AGENT_WINDOWS|.zip.sha512>`__)         |
++--------------+---------+--------------+---------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------+
+|    macOS     |  |WAZUH_LATEST|  |     64bit    | `wazuh-agent-macos-amd64-debug-info-|WAZUH_LATEST|-|WAZUH_REVISION_WPK_AGENT_MACOS|.tar.gz <https://packages.wazuh.com/4.x/wpk/macos/x86_64/pkg/wazuh-agent-macos-amd64-debug-info-|WAZUH_LATEST|-|WAZUH_REVISION_WPK_AGENT_MACOS|.tar.gz>`_ (`sha512 <https://packages.wazuh.com/|CURRENT_MAJOR|/checksums/wazuh/|WAZUH_LATEST|/wazuh-agent-macos-amd64-debug-info-|WAZUH_LATEST|-|WAZUH_REVISION_WPK_AGENT_MACOS|.tar.gz.sha512>`__)   |
++--------------+---------+--------------+---------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------+
```

Comment:
Despite some of the WPK packages use raw URLs, try to create the URL based on variables (change 4.x to CURRENT_MAJOR)